public-ui/src/app/data-browser/cdr-version/cdr-version-info.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../data-browser/base-react/base-react.wrapper';
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const containerElementName = 'root';
+
+export const style = `
+.result-body-item {
+    padding: 6px 0;
+}
+.cdr-info {
+    font-size: 14px;
+}
+`;
+
+interface State {
+    numParticipants: number;
+    creationTime: Date;
+}
+
+export class CdrVersionReactComponent extends React.Component<{}, State> {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+        numParticipants: '',
+        creationTime: '',
```

Comment:
I think if you init these with `null` or `undefined`, it should fix the ui error